test/specs/commonTests/implementsCollectionShorthandProp.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
+import { mount } from './isConformant'
+import * as _ from 'lodash'
+import { DefaultShorthandTestOptions, ShorthandTestOptions } from './implementsShorthandProp'
+
+export default Component => {
+  return function implementsCollectionShorthandProp(
+    shorthandPropertyName: string,
+    ShorthandComponent: React.ComponentType,
+    options: ShorthandTestOptions = DefaultShorthandTestOptions,
+  ) {
+    const { mapsValueToProp } = options
+
+    describe(`shorthand property for '${ShorthandComponent.displayName}'`, () => {
+      test(`is defined`, () => {
+        expect(Component.propTypes[shorthandPropertyName]).toBeTruthy()
+      })
+
+      test(`array of string values are handled as ${
```

Comment:
grammar nit: 'as' -> 'is' :)

Reply:
Agreed, it is better explained that way!